Added new background for splash screen and a new logo

Gaganindoriya opened the pull request:
I make the splash logo as static as said by @mohancm

iamkunalpitale committed:
Splash Screen Coding

iamkunalpitale committed:
Splash Screen XML

prafulnayak committed:
Succefully 4 Sqlite Table created and Tested

prafulnayak committed:
Contract classs(QuizContract), DbHelper Class(QuizDbHelped) and Content Provider(QuizProvider) designed.

coding and testing for query methed and insert method for all 4 Sqlite Table done.

All with dummy data.

prafulnayak committed:
CursorLoader Implemented. It will retribe all subscribed lesson in Lesson Screen.

All the lessons are stored in SQLite.

Gaganindoriya committed:
1changed code of activity_splash.xml
2-changed code of SplashActivity.java
3-Added spalshlogo.png image
4-adde anim alpha.xml and translate.xml files under res/anim directory
5-Added one more color to color.xml(sky blue)

Gaganindoriya committed:
1changed code of activity_splash.xml
2-changed code of SplashActivity.java
3-Added spalshlogo.png image
4-adde anim alpha.xml and translate.xml files under res/anim directory
5-Added one more color to color.xml(sky blue)

mohancm commented:
@Gaganindoriya  this your commits have conflicts.
delete old repo  and fork again, make your changes and push it

Gaganindoriya committed:
1-Changed The Splash screen background to match the app theme(Black and yellow)
2- Replace Splash screen logo to more expressive and 3d looking

3-changed the splash animation time from 2000 to 2500 ms

Gaganindoriya committed:
1-make splash logo static
